dolphinscheduler-worker/src/main/bin/start.sh

```diff
@@ -21,7 +21,6 @@ DOLPHINSCHEDULER_HOME=${DOLPHINSCHEDULER_HOME:-$(cd $BIN_DIR/..; pwd)}
 
 source "$DOLPHINSCHEDULER_HOME/conf/dolphinscheduler_env.sh"
 
-chmod -R 700 ${DOLPHINSCHEDULER_HOME}/conf
```

Comment:
chould we change to `755`? WDYT

Reply:
> There is account password information in the application.yaml file in the conf directory, so it is not recommended to set the permission to 755.
> 
> If it is set to `chmod -R 755 ${DOLPHINSCHEDULER_HOME}/conf/dolphinscheduler_env.sh`, when the conf folder does not have other user execution permissions, it will still report that the dolphinscheduler_env.sh file permissions are insufficient.
> 
> Do you have any good suggestions for only setting dolphinscheduler_env.sh file permissions to 755 and other file permissions to 700?

IMHO, I have the following two ideas：
1. Just keep only one `dolphinscheduler_env.sh` in bin/env/dolphinscheduler_env.sh, all services no longer keep a separate copy of the file. Becasue all these files are same. So we just need chmod -R 755 `bin/env/dolphinscheduler_env.sh`
2. All tmp execution path source a source file in other disk path is not a good idea. May can copy it to execution path and clear it after task finished. This ensures that all files needed for the task to run are stored in the temporary execution directory. WDYT.

Reply:
Remove `chmod -R 700 ${DOLPHINSCHEDULER_HOME}/conf` is ok for me.

If fact, `${DOLPHINSCHEDULER_HOME}/conf/dolphinscheduler_env.sh` is the default env faile will be sourced by task, we can change this in common.properties `dolphinscheduler.env.path`.